Dockerfile

```diff
@@ -1,9 +1,17 @@
+# temporary image: build an xsnippet-api wheel and collect all its dependencies
 FROM python:latest
-MAINTAINER The XSnippet Team <dev@xsnippet.org>
 
 COPY . /app
-RUN pip install /app && rm -rf /app
+WORKDIR /app
+RUN mkdir dist && pip wheel --wheel-dir dist .
```

Comment:
I'm just curious, would it download all dependencies we have into `diet`?

Reply:
Yes, that's correct. I've added a `ls -la dist` call for visibility in build logs.